### ISSUE TITLE
Support for .ts files

### ISSUE THREAD

**Issue by kieranju** [samuelmeuli/glance#3]
It would be really nice to see support for TypeScript (*.ts) files.

I might take a stab at it if I find some spare time.

I'd also like to say, this is a great idea to keep an up-to-date collection of Quick Look plugins bundled together. Nice work!

**Comment by samuelmeuli** [comment]
Thanks!

As far as I can tell, this isn't currently possible. macOS recognizes `.ts` files as media files:

```
❯ mdls -name kMDItemContentType -name kMDItemContentTypeTree /path/to/file.ts
kMDItemContentType     = "public.mpeg-2-transport-stream"
kMDItemContentTypeTree = (
    "public.item",
    "public.movie",
    "public.audiovisual-content",
    "public.mpeg-2-transport-stream",
    "public.data",
    "public.content"
)
```

And unfortunately, it seems like macOS doesn't allow us to handle `public.mpeg-2-transport-stream` UTIs. Same with `.xml` and `.plist` files.

**Comment by samuelmeuli** [comment]
I've added support for `.tsx` files in 72333191686f6caf9561afef7a25eae8f61a183f.

**Comment by samuelmeuli** [comment]
`.ts` support released in [v1.1.0](https://github.com/samuelmeuli/glance/releases/tag/v1.1.0).

**Comment by bengry** [comment]
> Thanks!
> 
> As far as I can tell, this isn't currently possible. macOS recognizes `.ts` files as media files:
> 
> ```
> ❯ mdls -name kMDItemContentType -name kMDItemContentTypeTree /path/to/file.ts
> kMDItemContentType     = "public.mpeg-2-transport-stream"
> kMDItemContentTypeTree = (
>     "public.item",
>     "public.movie",
>     "public.audiovisual-content",
>     "public.mpeg-2-transport-stream",
>     "public.data",
>     "public.content"
> )
> ```
> 
> And unfortunately, it seems like macOS doesn't allow us to handle `public.mpeg-2-transport-stream` UTIs. Same with `.xml` and `.plist` files.

Could this be worked around using a root user or something similar? I'm sure most people using this app wouldn't mind turning off the default ".ts" preview, as the format isn't used very often anymore.
